docs/examples/how-tos.md

```diff
@@ -0,0 +1,396 @@
+
+// Get the editor instance from the editable element.
+const editorInstance = domEditableElement.ckeditorInstance;
+
+// Use the editor instance API.
+editorInstance.setData( '<p>Hello world!<p>' );
+```
+
+### How to list all instances of the editor?
+
+By default, CKEditor 5 has no global registry of editor instances. But if necessary, such a feature can be easily implemented, as explained in this [Stack Overflow answer](https://stackoverflow.com/a/48682501/1485219).
+
+## Editor's API
+
+### How to insert some content into the editor?
+
+Because CKEditor 5 uses a custom {@link framework/architecture/editing-engine data model}, whenever you want to insert anything, you should modify the model first, which is then converted back to the view where the users input their content (called "editable"). In CKEditor 5, HTML is just one of many possible output formats. You can learn more about the ways of changing the model in the {@link framework/architecture/editing-engine#changing-the-model dedicated guide}.
+
+For example, to insert a new link at the current position, use the following snippet:
+
+```js
+editor.model.change( writer => {
+	const insertPosition = editor.model.document.selection.getFirstPosition();
+
+	writer.insertText(
+		'CKEditor 5 rocks!',
+		{ linkHref: 'https://ckeditor.com/' },
+		insertPosition
+	);
+} );
+```
+
+And to insert some plain text, you can use a slightly shorter one:
+
+```js
+editor.model.change( writer => {
+	writer.insertText(
+		'Plain text',
+		editor.model.document.selection.getFirstPosition()
+	);
+} );
+```
+
+You may have noticed that a link is represented as a text with an attribute in the editor model. See the API of the {@link module:engine/model/writer~Writer model writer} to learn about other useful methods that can help you modify the editor model.
+
+To insert some longer HTML code, you can parse it to the {@link module:engine/model/documentfragment~DocumentFragment model fragment} first and then {@link module:engine/model/model~Model#insertContent insert} it into the editor model:
+
+```js
+const content =
+	'<p>A paragraph with <a href="https://ckeditor.com">some link</a>.</p>';
+const viewFragment = editor.data.processor.toView( content );
+const modelFragment = editor.data.toModel( viewFragment );
+
+editor.model.insertContent( modelFragment );
+```
+
+### How to focus the editor?
+
+```js
+// Focus the editor.
+editor.focus();
+
+// Focus the editing area of the editor.
+editor.editing.view.focus();
+```
+
+### How to delete selected blocks?
+
+```js
+const selectedBlocks = Array.from(
+	editor.model.document.selection.getSelectedBlocks()
+);
+const firstBlock = selectedBlocks[ 0 ];
+const lastBlock = selectedBlocks[ selectedBlocks.length - 1 ];
+
+editor.model.change( writer => {
+	const range = writer.createRange(
+		writer.createPositionAt( firstBlock, 0 ),
+		writer.createPositionAt( lastBlock, lastBlock.maxOffset )
+	);
+
+	writer.remove( range );
+} );
+```
```

Comment:
As for inserting content into the editor, I'd suggest using `model.deleteContent()`.

Reply:
Btw this snippet is not removing the first and last selected block but the title is mentioning this.

Reply:
> As for ~inserting~ deleting content in the editor (...) Btw this snippet is not removing the first and last selected block but the title is mentioning this.

@niegowski what's the correct way?